web/src/pages/api/auth0.ts

```diff
@@ -0,0 +1,2 @@
+import { auth0Handler } from "src/api/auth0";
```

Comment:
we keep it in src/api because we use graphql. To use graphql we use codgen. And codgen configured on this folder.
I think I can update it to generate hooks for graphql in pages/api, but I don't think it's a good idea to keep gql files in pages directory.